run-local-testnet/action.yaml

```diff
@@ -25,6 +25,21 @@ inputs:
 runs:
   using: composite
   steps:
+    # There is no way to configure post-if with this, so we just always print the logs
+    # in the post step.
+    - name: Set up printing logs in a post step
+      uses: ./with-post-step
```

Comment:
There's no guarantee that the calling job has this path since it's unlikely to have cloned the entire `aptos-labs/actions` repo, so you'll likely need to do `aptos-labs/actions/with-post-step@main`.